scripts/pr-infra-compose.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# This script leverages the host override docker-compose file to run the OSRD stack.
+set -e
+
+# Usage
+
+if [ "$#" -ne 3 ] ||
+    { [ "$3" != "up" ] && [ "$3" != "down" ]; }; then
+    echo "Usage: $0 pr-number osrd.backup (up | down)" >&2
+    exit 1
+fi
+
+# Change ports in needed files
+sed -i "s/{{pr-nb}}/pr-$1/g" ./docker/docker-compose.pr-tests.yml
+sed -i "s/{{pr-nb}}/pr-$1/g" ./docker/osrdyne-pr-tests.yml
```

Comment:
Can we do this via environment variables? It would be much better.

**If not**, we shouldn't modify these files in place:

- It creates unwanted changes in the working tree
- Multiple invocation of `pr-infra-compose.sh` will fail once one replacement is done (unless people roll it back, but isn't the goal to make PR testing as seamless as possible?)

We should instead move `docker/docker-compose.pr-tests.yml` to  `docker/docker-compose.pr-tests.template.yml`, add `/docker/docker-compose.pr-tests.yml` to the `.gitignore`, and have sed generate it. (Ditto for `docker/osrdyne-pr-tests.yml`)

Reply:
Env variables can be used on this.

Reply:
I could switch to env vars for everything but not for the osrdyne.yml config file unfortunately

Reply:
Thanks to Khoyo I was able to pass it using an env var too.